atlassian/modules.go

```diff
@@ -0,0 +1,27 @@
+package atlassian
+
+import (
+	"github.com/amp-labs/connectors/common/paramsbuilder"
+)
+
+// Atlassian's modules can include CloudID variable.
+// Token discovery API will be called to substitute CloudID.
+
+// ModuleJira is the module used for accessing standard CRM objects.
+var ModuleJira = paramsbuilder.APIModule{ // nolint: gochecknoglobals
+	Label:   "ex/jira/{{.cloudID}}/rest/api",
```

Comment:
@RajatPawar do you think the having a module with variables is a good approach?

Reply:
Can this be a part of the base URL? I'd like us to do modules a little more comprehensively ideally. In this particular case, if it becomes a part of the base URL, it just is a workspace var and can behave like docusign.

Reply:
I am struggling with balancing 2 things. Jira APIs and whatever should come in like Confluence. I would honestly introduce modules only after working on the second deep connector.

I was thinking that BaseURLs are shared among 2 while each has its own structure one could be `ex/jira` the other something else. Checking [Confluence docs](https://developer.atlassian.com/cloud/confluence/rest/v2/intro/#auth) it seems that base URL differs entierly.

I could keep `rest/api/3` as module for now.

Reply:
It is a bit confusing to have `WithModule(.)` if there is only 1 or raw aka empty. Honestly, all connectors fall under this category. Only some will have `{ empty, defaultVersion, option[1...many] }.`